src/sentry/lang/javascript/processing.py

```diff
@@ -116,6 +140,8 @@ def process_payload(data):
     if not _handle_response_status(data, response):
         return data
 
+    data.setdefault("errors", []).extend(map_symbolicator_process_js_errors(response.get("errors")))
```

Comment:
The `setdefault` though would probably violate the `assert "errors" not in event.data`?
Since now at least the array always exists?

Reply:
Good catch. Updated.